frontend/device/android/device.lua

```diff
@@ -244,6 +245,7 @@ function Device:init()
         android.setBackButtonIgnored(true)
     end
 
+    -- TODO fl_last_level is a leftover from frontlightwidget
```

Comment:
This comment is right now. The old code can be removed from frontlightwidget. I think this can be done later and now just comment the lines below.

Reply:
OK.